contrib/babelfishpg_tsql/sql/object_definition_tsql.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE sys.babelfish_view_def (
+	db_name NAME NOT NULL,
+	schema_name NAME NOT NULL,
+	object_name NAME NOT NULL,
+	definition TEXT NOT NULL COLLATE "C",
```

Comment:
Let's use `nvarchar(max)` to align with the definition of `sys.sql_modules` (https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/system-catalog-views/sys-all-sql-modules-transact-sql.md). Btw, Is there any reason to use C collation here?

Reply:
Even if we use nvarchar(max) we might need to cast it to different string types of different lengths while using it, since sys.sql_modules is not the only system view which stores the definitions

Reply:
I'm afraid of using pg-only types (NAME, TEXT) in BBF catalogs because they behave differently, e.g., ANSI_PADDINGS. And would you point me to some examples where different system views use different data types for the object definitions or names?

Reply:
Fixed. Used TSQL types in catalog definition.